src/web/src/components/SearchInput/SearchInput.tsx

```diff
@@ -1,11 +1,23 @@
 import PostSearchInput from './PostSearchInput';
 import AuthorSearchInput from './AuthorSearchInput';
+import DateSearchInput from './DateSearchInput';
 import useSearchValue from '../../hooks/use-search-value';
 
 const SearchInput = () => {
   const { filter } = useSearchValue();
 
-  return filter === 'author' ? <AuthorSearchInput /> : <PostSearchInput />;
```

Comment:
Let's rebase to pick up the change that @drew5494 just added to fix the search autofocus problem.  I don't want to re-introduce it.

As an aside (@DukeManh), the fix for that feels really brittle, and easy to break again.  We probably need a comment to help future devs not regress.